feat(open-pr-comments): docs (#8883)
getsentry/sentry-docs: src/docs/product/integrations/source-code-mgmt/github/index.mdx

### Get Sentry Comments on Pull Requests

Sentry can comment on merged and open pull requests. These features are automatically enabled once your GitHub integration has been set up. To disable it, go to **Settings > Integrations > GitHub** and toggle it off.

![How to disable Sentry comments on GitHub pull requests](github-org-settings.png)

#### Merged Pull Request Comments

If a recently merged pull request is suspected of causing an issue, Sentry will comment on it to surface the issue, as shown in the example below.

![Sentry comment on suspect pull request in GitHub](sentry-github-pr-comment.png)

Sentry will only comment on pull requests less than two weeks old that are associated with suspect commits. The comment will continuously be updated with up to five issues the pull request is suspected to have caused.

#### Open Pull Request Comments

<Include name="feature-available-for-user-group-early-adopter.mdx" />

> When a pull request is opened, Sentry will parse the filenames and function names modified in the pull request from the PR diff to find recent unhandled, unresolved issues associated with your pull request.

Reviewer: ```suggestion
When a pull request is opened, Sentry will parse the filenames and function names modified to find recent error issues associated with the code changes.
```
Looks like this got missed?

Author: oh oops, thanks!

> ### Get Sentry Comments on Suspect Pull Requests

Author: ```suggestion
### Get Sentry Comments on Pull Requests
```

> These features will be automatically enabled for you once your GitHub integration has been set up. To disable it, go to **Settings > Integrations > GitHub** and toggle it off.

Author: hmm how about "Sentry can comment on merged and open pull requests" since they do different things?

> These features will be automatically enabled for you once your GitHub integration has been set up. To disable it, go to **Settings > Integrations > GitHub** and toggle it off.

Author: where would you suggest putting this?

> These features will be automatically enabled for you once your GitHub integration has been set up. To disable it, go to **Settings > Integrations > GitHub** and toggle it off.

Reviewer: that's perfect!

> These features will be automatically enabled for you once your GitHub integration has been set up. To disable it, go to **Settings > Integrations > GitHub** and toggle it off.

Reviewer: Oh, this was an old comment. I think the updated header clarifies this now so I'll go ahead and resolve this

![How to disable Sentry comments on GitHub pull requests](github-org-settings.png)

#### Merged Pull Request Comments

If a recently merged pull request is suspected of causing an issue, Sentry will comment on it as in the example below.

![Sentry comment on suspect pull request in GitHub](sentry-github-pr-comment.png)

Sentry will only comment on pull requests less than two weeks old that are associated with suspect commits. The comment will continuously be updated with up to five issues the pull request is suspected to have caused.

### Open Pull Request Comments

> When a pull request is opened, Sentry will parse the filenames and function names modified in the pull request from the PR diff to find recent unhandled, unresolved issues associated with your pull request.

Author: it's only errors for now... but i can modify this

> When a pull request is opened, Sentry will parse the filenames and function names modified in the pull request from the PR diff to find recent unhandled, unresolved issues associated with your pull request.

Reviewer: Oh, it should be modified in the UI also then!

> When a pull request is opened, Sentry will parse the filenames and function names modified in the pull request from the PR diff to find recent unhandled, unresolved issues associated with your pull request.

Author: true, thanks!
